subsystem/kubeapi_test.go

```diff
@@ -1167,6 +1183,73 @@ var _ = Describe("[kube-api]cluster installation", func() {
 		checkAgentCondition(ctx, host.ID.String(), v1beta1.BoundCondition, v1beta1.UnboundReason)
 	})
 
+	It("Agent labels", func() {
+		By("Deploy InfraEnv")
+		infraEnvSpec.ClusterRef = nil
+		deployInfraEnvCRD(ctx, kubeClient, infraNsName.Name, infraEnvSpec)
+
+		infraEnvKubeName := types.NamespacedName{
+			Namespace: Options.Namespace,
+			Name:      infraNsName.Name,
+		}
+
+		By("Verify ISO URL is populated")
+		Eventually(func() string {
+			return getInfraEnvCRD(ctx, kubeClient, infraEnvKubeName).Status.ISODownloadURL
+		}, "15s", "5s").Should(Not(BeEmpty()))
+
+		By("Register Agent to InfraEnv")
+		infraEnvKey := types.NamespacedName{
+			Namespace: Options.Namespace,
+			Name:      infraNsName.Name,
+		}
+		infraEnv := getInfraEnvFromDBByKubeKey(ctx, db, infraEnvKey, waitForReconcileTimeout)
+		configureLocalAgentClient(infraEnv.ID.String())
+		host := &registerHost(*infraEnv.ID).Host
+		hwInfo := validHwInfo
+		hwInfo.Interfaces[0].IPV4Addresses = []string{defaultCIDRv4}
+		generateHWPostStepReply(ctx, host, hwInfo, "hostname1")
+
+		By("Verify agent inventory labels")
+		key := types.NamespacedName{
+			Namespace: Options.Namespace,
+			Name:      host.ID.String(),
+		}
+		var agentLabels map[string]string
+		Eventually(func() bool {
+			agent := getAgentCRD(ctx, kubeClient, key)
+			agentLabels = agent.GetLabels()
+			_, ok := agentLabels["inventory.agent-install.openshift.io/storage-hasnonrotationaldisk"]
+			return ok
+		}, "30s", "10s").Should(BeTrue())
```

Comment:
Lets make the interval every second, it will make the test run faster 
Relevant for all the intervals

Reply:
Done